src/v/cluster/node/types.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#pragma once
+
+#include "model/fundamental.h"
+#include "model/metadata.h"
+#include "model/timestamp.h"
+#include "reflection/adl.h"
+#include "types.h"
+#include "utils/human.h"
+#include "utils/named_type.h"
+
+#include <seastar/core/sstring.hh>
+
+#include <fmt/ostream.h>
+
+namespace cluster::node {
+
+//   _   _           _            _                 _       _        _
```

Comment:
Nit, maybe I'm just being grumpy, but can we avoid the ascii-art for text in code comments?  I have to squint a bit to read this and it pushes the rest of the content down my screen.

Reply:
I'm not attached to it and was curious what reaction I'd get. Last gig used these for section headers and it grew on me. I'll remove it in a subsequent patch.

Reply:
![image](https://user-images.githubusercontent.com/558234/150477280-399c6313-a0fd-476b-938d-51c6b9920c5a.png)
Github needs to fix line spacing.. it got butchered on here. :laughing: